goal_thr_x: 1.0
goal_thr_y: 1.0
goal_thr_z: 1.0
goal_thr_yaw: 0.3
px: 0.5    
dx: 0.4    
ix: 0.2    
py: 0.5    
dy: 0.4    
iy: 0.2    
max_xy_v: 5.7    
pz_d: 0.8    
dz_d: 0.1    
iz_d: 0
pz_a: 0.8
dz_a: 0.2
iz_a: 0.0    
max_vz: 2.6    
p_yaw: 0.4    
d_yaw: 0.2    
i_yaw: 0
max_vyaw: 1.5   
small_xyz_v: 0.5
